request.go

```diff
@@ -244,6 +244,19 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 				continue
 			}
 
+			if v.Kind() == reflect.String {
+				stringValue := v.Interface().(string)
+
+				// The field may or may not be a pointer to a string; handle both cases
+				if fieldValue.Kind() == reflect.Ptr {
+					fieldValue.Set(reflect.ValueOf(&stringValue))
```

Comment:
So, I looked over the code here again. Put together a [playground](https://play.golang.org/p/QmJtDa6a7X) to test it. And it looks like what is being done here is proper.

If the user does pass in a value that is a string, then this logic will attempt to assign it as the appropriate `string` or `*string`. If the user passes nothing in for the field, then it will simply be left `nil` on the final struct.

**HOWEVER:** this logic does not account for bad types being passed in by the user. See #38. This code assumes that if the user passes in a string, then the struct field **MUST** be either `string` or `*string`, which is simply not always going to be the case. Make sense?

@aren55555 I have some code in #38 to deal with type errors. Want to grab some of the code there and slap it in place for this? @shwoodard what do you think?